easybuild/easyconfigs/i/IMB/IMB-4.1_make_mpich.patch

```diff
@@ -0,0 +1,11 @@
+Prevent: Variable MPI_HOME="" does not seem to contain a valid mpicc.
+Author: Fokke Dijkstra (University of Groningen)
+--- src/make_mpich	2015-02-16 13:31:11.838082386 +0100
++++ src/make_mpich	2015-02-16 13:40:19.000844455 +0100
+@@ -1,5 +1,5 @@
+ # Enter root directory of mpich install
+-MPI_HOME=
++#MPI_HOME=
```

Comment:
@RvDijk  hmm, it seems to me this patch may not be required at all if you make EasyBuild run this:

```
make -f make_mpich MPI_HOME=$EBROOTOPENMPI
```

via `buildopts`:

```python
buildopts = "-f make_mpich MPI_HOME=$EBROOTOPENMPI"
```

Can you verify that, and if it works, just drop this patch and go with that approach?
Less patches means spending less time maintaining them... ;)